www/src/pages/en/deployment/docker.md

```diff
@@ -4,187 +4,195 @@ description: Deployment with Docker
 layout: ../../../layouts/docs.astro
 ---
 
-You can containerize this stack and deploy it as a single container using Docker, or as a part of a group of containers using docker-compose.
+You can containerize this stack and deploy it as a single container using Docker, or as a part of a group of containers using docker-compose. See [`ct3a-docker`](https://github.com/ajcwebdev/ct3a-docker) for an example repo based on this doc.
 
-Please note that Next.js requires a different process for buildtime (available in the frontend, prefixed by `NEXT_PUBLIC`) and runtime environment, server-side only, variables. In this demo we are using two variables, `DATABASE_URL` (used by the server) and `NEXT_PUBLIC_CLIENTVAR` (used by the client). Pay attention to their positions in the `Dockerfile`, command-line arguments, and `docker-compose.yml`.
-
-1.  In your [next.config.mjs](https://github.com/t3-oss/create-t3-app/blob/main/cli/template/base/next.config.mjs), add the `standalone` output-option to your config:
+## Docker Project Configuration
 
-    ```diff
-      export default defineNextConfig({
-        reactStrictMode: true,
-        swcMinify: true,
-    +   output: "standalone",
-      });
-    ```
+Please note that Next.js requires a different process for build time (available in the frontend, prefixed by `NEXT_PUBLIC`) and runtime environment, server-side only, variables. In this demo we are using two variables, pay attention to their positions in the `Dockerfile`, command-line arguments, and `docker-compose.yml`:
 
-2.  Remove the `env`-import from [next.config.mjs](https://github.com/t3-oss/create-t3-app/blob/main/cli/template/base/next.config.mjs):
-
-    ```diff
-    - import { env } from "./src/env/server.mjs";
-    ```
-
-3.  Create a `.dockerignore` file with the following contents:
+- `DATABASE_URL` (used by the server)
+- `NEXT_PUBLIC_CLIENTVAR` (used by the client)
 
-    <details>
+### 1. Next Configuration
 
-    &lt;summary&gt;dockerignore&lt;/summary&gt;
+In your [`next.config.mjs`](https://github.com/t3-oss/create-t3-app/blob/main/cli/template/base/next.config.mjs), add the `standalone` output-option configuration to [reduce image size by automatically leveraging output traces](https://nextjs.org/docs/advanced-features/output-file-tracing):
 
-    ```
-    .env
-    Dockerfile
-    .dockerignore
-    node_modules
-    npm-debug.log
-    README.md
-    .next
-    .git
-    ```
+```diff
+// next.config.mjs
 
-  </details>
+export default defineNextConfig({
+  reactStrictMode: true,
+  swcMinify: true,
++ output: "standalone",
+});
+```
 
-4.  Create a `Dockerfile` with the following contents:
+### 2. Remove Env Import
 
-    <details>
+Remove the `env`-import from [`next.config.mjs`](https://github.com/t3-oss/create-t3-app/blob/main/cli/template/base/next.config.mjs):
```

Comment:
we've been asked why you need to do this so maybe we should add something here regarding this being pulled in to the build image or smth 🤔